packages/react-core/src/components/Page/examples/Page.md

```diff
@@ -62,9 +62,9 @@ When the `isManagedSidebar` property is true, it manages the sidebar open/close
 
 ### Filled page sections
 
-By default, the last page section is "filled", meaning it fills the available vertical space of a page.
+By default, page sections will not fill available vertical space of a page.
 
-To change the default behavior, use the `isFilled` property. To make other page sections "filled", set `isFilled` equal to "true". To disable the last page section from being "filled", set `isFilled` equal to "false".
+To change the default behavior, pass `isContentFilled` to the `Page` component, and use the `isFilled` property on any child `PageSection` or `PageGroup` components to specify whether they should fill the available vertical space.
```

Comment:
```suggestion
To change this default behavior, pass `isContentFilled` to the `<Page>` component and `isFilled={true}` to any child `<PageSection>` or `<PageGroup>` component that you want to fill the space. If multiple components are set to fill, then the available space will be split equally between them.
```
I see the example uses isFilled={true} but it also looks like just isFilled on its own works? If it's accurate, we could instead just say "pass isFilled to any child...", but I'll leave that up to you.

Also mentioned splitting between multiple items, but that can be removed if its not a typical use case